instrument/D2B_Definition.xml

```diff
@@ -14,17 +14,33 @@ name="D2B" valid-from="1900-01-31 23:59:59" valid-to="2100-01-31 23:59:59" last-
       <handedness val="right" />
     </reference-frame>
   </defaults>
-  <component type="moderator">
-    <location z="-2" />
+  <component type="monochromator">
```

Comment:
The changes for D2B are not mentioned in the PR. Can the generator scripts be updated, and a similar doc as for the others be created? This could go in a separate PR.